reporters_db/data/reporters.json

```diff
@@ -15732,7 +15719,6 @@
             "name": "McGrath's Mandamus Cases (Michigan)",
             "notes": "Also known as 'Mandamus cases decided in the Supreme court of Michigan'",
             "variations": {
-                "M.,": "McGrath",
```

Comment:
why are you removing this?

Reply:
I found some citations with incorrect reporter due to a variation, we have citations like "106 McGrath 42" but McGrath is a nominative reporter, it only has one volume, the citation from original source is "106 Mich. 42"

CL with 106 McGrath 42:  https://www.courtlistener.com/opinion/9288493/obrien-v-circuit-judge/

caselaw pdf for 106 Mich. 42: https://static.case.law/mich/106/case-pdfs/0042-01.pdf 

M. refers to Michigan Reports.

Reply:
okay.